django/contrib/admin/templates/admin/base.html

```diff
@@ -108,9 +108,9 @@
           <br class="clear">
         </div>
         <!-- END Content -->
-        {% block footer %}<div id="footer"></div>{% endblock %}
       </main>
     </div>
+    {% block footer %}<footer id="footer"></footer>{% endblock %}
```

Comment:
I thought we [agreed](https://code.djangoproject.com/ticket/35115#comment:6) on a different approach :thinking: 
```suggestion
    <footer id="footer">{% block footer %}{% endblock %}</footer>
```

Reply:
I'm sorry, I missed Thibaud's +1 on this decision. I disagree on this method for two reasons:

1. It makes the footer element itself not overridable.
2. The other elements, like the page header, also have the element defined within the block, so it feels more consistent to keep it that way.

Should we put this up for discussion here on in the ticket?

Reply:
> It makes the footer element itself not overridable.

So, exactly the same as situation as with `<main>`.

Reply:
I put the block tags inside the element.

Reply:
We can ask @thibaudcolas for the confirmation, if you want.

Reply:
It's OK, either solution has pros and cons, I can understand enforcing the footer tags like this!